internal/cephfs/core/metadata.go

```diff
@@ -19,8 +19,8 @@ package core
 import (
 	"errors"
 	"fmt"
-	"strings"
 
+	libceph "github.com/ceph/go-ceph/cephfs"
```

Comment:
consider renaming to `libcephfs`, as `libceph` could be for anything (but does not exist)

Reply:
Ah, yes. Missed it. 
Thanks!